app/views/posts/show.html.erb

```diff
@@ -15,6 +15,14 @@
     <%= button_to "Destroy this post", @post, method: :delete, class: "btn btn-danger", data: { confirm: "Are you sure you want to delete this post?" } %>
   <% end %>
 </div>
+<% @post.replies.each do |reply| %>
+  <div class="reply">
+    <p><%= reply.content %></p>
+    <p>Score: <%= reply.score %></p>
+    <%= button_to 'Upvote', upvote_reply_path(reply), method: :post, class: "upvote-button" %>
```

Comment:
<html>
<body>
<!--StartFragment--><p style="font-family: helvetica, verdana, arial, sans-serif; font-size: 13px; line-height: 18px; color: rgb(236, 236, 236); font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(34, 34, 34); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Showing<span> </span><i>/mnt/c/Users/elida/Computer Science/software-engineering/Stack-Overflow-Clone/app/views/posts/show.html.erb</i><span> </span>where line<span> </span><b>#22</b><span> </span>raised:</p><pre style="font-size: 11px; white-space: pre-wrap; color: rgb(236, 236, 236); font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(34, 34, 34); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><code>undefined method `upvote_reply_path' for an instance of #&lt;Class:0x00007efdc8b8caa0&gt;</code></pre><div class="source " id="frame-source-0-0" style="border: 1px solid rgb(85, 85, 85); background: rgb(51, 51, 51); max-width: 978px; color: rgb(236, 236, 236); font-family: helvetica, verdana, arial, sans-serif; font-size: 13px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="info" style="margin: 0px; padding: 0.5em;">Extracted source (around line<span> </span><strong style="margin: 0px; padding: 0px;">#22</strong>):</div><div class="data" style="margin: 0px; padding: 0px; font-size: 10.4px; overflow: auto; background: rgb(68, 68, 68);">
20 21 22 23 24 25 | <p><%= reply.content %></p>     <p>Score: <%= reply.score %></p>     <%= button_to 'Upvote', upvote_reply_path(reply), method: :post, class: "upvote-button" %>     <%= button_to 'Downvote', downvote_reply_path(reply), method: :post, class: "downvote-button" %>   </div> <% end %>
-- | --


</div></div><p style="font-family: helvetica, verdana, arial, sans-serif; font-size: 13px; line-height: 18px; color: rgb(236, 236, 236); font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(34, 34, 34); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"></p><p style="font-family: helvetica, verdana, arial, sans-serif; font-size: 13px; line-height: 18px; color: rgb(236, 236, 236); font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(34, 34, 34); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><code>Rails.root: /mnt/c/Users/elida/Computer Science/software-engineering/Stack-Overflow-Clone</code></p><!--EndFragment-->
</body>
</html>